monkey/infection_monkey/utils/plugins/load_plugins.py

```diff
@@ -0,0 +1,45 @@
+import importlib
+import inspect
+import logging
+from os.path import dirname, basename, isfile, join
+import glob
+
+from infection_monkey.utils.plugins.plugin import Plugin
+
+LOG = logging.getLogger(__name__)
+
+
+def _get_candidate_files(base_package_file):
+    files = glob.glob(join(dirname(base_package_file), "*.py"))
+    return [basename(f)[:-3] for f in files if isfile(f) and not f.endswith('__init__.py')]
+
+
+def get_instances(base_package_name, base_package_file, parent_class: Plugin):
```

Comment:
Not a very descriptive name. I'd rather elaborate to at least `get_class_instances`. Also, it would be best if this was in Plugin class as class method instead of some separate helper file. Then we could do `PBA.get_class_instances(infection_monkey.post_breach.actions.__package__, infection_monkey.post_breach.actions.__file__)`. This is more readable (plugin code in plugin class, not some utils file somewhere) and we don't need to pass `parent_class` manually. 
To take it even further `Plugin` class should have `base_package_name` and `base_package_file` abstract properties. This way we know that each `Plugin` child implements these values and we can use the same `get_class_instances` method on all `Plugin` child. This would also make the call to method `PBA.get_class_instances()` even more readable.